www/components/Function.tsx

```diff
@@ -27,16 +34,32 @@ const Container = styled(Box)`
 const FuncDocs = styled(Documentation)`
   margin-left: 1em;
 `;
+const Keyword = styled.span`
+  font-weight: bold;
+  color: rgb(249, 38, 114);
```

Comment:
Ideally we'd load these colors from some theme, but we'll want to inject links etc., so just generating the code string and running it through a highlighter is not really an option.